src/ng-select/ng-select.component.ts

```diff
@@ -113,6 +113,15 @@ export class NgSelectComponent implements OnInit, OnDestroy, OnChanges, AfterVie
     get single() {
         return !this.multiple;
     }
+    @Output('loadingChange') loadingChange: EventEmitter<Boolean> = new EventEmitter<Boolean>();
+    get loading(): boolean {
+        return this.isLoading;
+    }
+    @Input()
+    set loading(value: boolean) {
```

Comment:
It would be great if you can update README.md and add these new API input and output to documentation.